Mailbox2.ino

```diff
@@ -104,6 +109,13 @@ void reconnect() {
       // Same for inside light
       mqttclient.publish("/outside/mailbox/inside/", "OFF");
       mqttclient.subscribe("/outside/mailbox/inside/set");
+
+      /* A-a-and same goes for the temp sensor
+       * Report a dummy value as a temp reading. The actual temp reading
+       * will occur in our main loop.
+       */
+      mqttclient.publish("/outside/mailbox/temperature/", "42.42");
+      mqttclient.subscribe("/outside/mailbox/temperature/set");
```

Comment:
This line would not be needed, it's only needed for variables that can be set from the home automation server. So above for the lights (/outside/mailbox/outside/ is a LED strip on the outside and can be set via MQTT) it's because we use the /set topic to actually change the status. In the case of temperature, we only need to report and will never get /set back.